### PR TITLE
Apply fix for ttml1/#201 (#358).

### PR DESCRIPTION
**Pull request by cconcolato**
partially addresses #358

### CODE REVIEW

**Comment by cconcolato** [comment]
I found the text strange too but did not want to deviate from the TTML1 change. 

@palemieux would you be ok to change TTML1's text and I would simultaneously change this PR?

File: spec/ttml2.xml [hunk 1]
```diff
@@ -18581,6 +18581,18 @@ agent with another agent that portrays the character.</p>
 <p>The <att>agent</att> attribute of a <el>ttm:actor</el> element must
 reference a <emph>significant</emph> <el>ttm:agent</el> element that denotes the agent
 acting the part of a character.</p>
+<p>An <el>ttm:actor</el> element referencing its parent <el>ttm:agent</el> element should be considered an error.</p>
```
Comment (palemieux) [inline spec/ttml2.xml]: In TTML2, we can say "is considered an error" or "must be considered an error" -- both forms are used.

**Comment by palemieux** [comment]
@cconcolato See review comment. TTML1 can probably say something along the lines of "An ttm:actor element should not reference its parent ttm:agent element."

**Comment by cconcolato** [comment]
@palemieux @skynavga I have updated the PR with the sentence you both suggested. Please review the PR again.

**Comment by palemieux** [comment]
@cconcolato I thought that TTML2 would be stronger and say _An ttm:actor element referencing its parent ttm:agent element is considered an error_, whereasTTML1 would say _An ttm:actor element should not reference its parent ttm:agent element_ . I do not however object to the current wording and approve the PR.

**Comment by skynavga** [comment]
@palemieux we can deprecate it formally, which is the most we can do at this time; @cconcolato let's add a deprecation here

**Comment by cconcolato** [comment]
@skynavga @palemieux I implemented what you agreed upon. Anyway, I'm fine with either using @palemieux latest proposal:
> An ttm:actor element referencing its parent ttm:agent element is considered an error.
or using @skynavga approach:
> An ttm:actor element referencing its parent ttm:agent element is deprecated.

Let me know what you prefer.

**Comment by cconcolato** [comment]
LGTM. @palemieux might want to align the text in TTML1.

File: spec/ttml2.xml [hunk 2]
```diff
@@ -18581,6 +18581,24 @@ agent with another agent that portrays the character.</p>
 <p>The <att>agent</att> attribute of a <el>ttm:actor</el> element must
 reference a <emph>significant</emph> <el>ttm:agent</el> element that denotes the agent
 acting the part of a character.</p>
+<p>The use of <phrase role="deprecated">a reference from the
+<att>agent</att> attribute of a <el>ttm:actor</el> element to its
+parent <el>ttm:agent</el> element is deprecated</phrase>, and,
+furthermore, the semantics of such a circular chain of references is
```
Comment (nigelmegitt) [inline spec/ttml2.xml]: typo: s/is/are

Comment (skynavga) [reply spec/ttml2.xml]: Fixed in https://github.com/w3c/ttml2/pull/632/commits/1d442cf379294b0382f6a2f7b03876adfbd49995 (to be merged).

**Pull request merged**